src/System.Data.SqlClient/src/System/Data/SqlClient/SqlParameter.cs

```diff
@@ -114,6 +114,37 @@ public SqlParameter(string parameterName, SqlDbType dbType, int size, string sou
             this.SourceColumn = sourceColumn;
         }
 
+        public SqlParameter(
+            string parameterName,
+            SqlDbType dbType, 
+            int size,
+            ParameterDirection direction,
+            byte precision,
+            byte scale,
+            string sourceColumn,
+            DataRowVersion sourceVersion,
+            bool sourceColumnNullMapping,
+            object value,
+            string xmlSchemaCollectionDatabase,
+            string xmlSchemaCollectionOwningSchema,
+            string xmlSchemaCollectionName
+        ) : this()
```

Comment:
Nit: the `: this()` is a little strange... seems like that could just be removed